Lib/test/test_math.py

```diff
@@ -377,6 +377,18 @@ def testAtan2(self):
         self.assertTrue(math.isnan(math.atan2(NAN, INF)))
         self.assertTrue(math.isnan(math.atan2(NAN, NAN)))
 
+    def testCbrt(self):
+        self.assertRaises(TypeError, math.cbrt)
+        self.ftest('cbrt(0)', math.cbrt(0), 0)
+        self.ftest('cbrt(1)', math.cbrt(1), 1)
+        self.ftest('cbrt(8)', math.cbrt(8), 2)
+        self.ftest('cbrt(27)', math.cbrt(27), 3)
+        self.ftest('cbrt(-1)', math.cbrt(-1), -1)
+        self.ftest('cbrt(-27)', math.cbrt(-27), -3)
+        self.assertEqual(math.cbrt(INF), INF)
```

Comment:
Please could you also add some tests for non-integral cases? It would be good to have a check that `cbrt(-0.0)` returns `-0.0` (with the correct sign), too.

Reply:
ok.

Reply:
But the test for `sqrt()` does not contain tests for non-integral cases.

Reply:
@serhiy-storchaka That's true, but I'm not sure why you're pointing it out in this PR discussion. Can you clarify? I'd be very happy to review a PR that fixed that defect for `math.sqrt`, but I think it's off-topic for _this_ PR.